wiki/Tournaments/Official_support/en.md

```diff
@@ -124,7 +124,7 @@ Registrant filtering, not to be confused with [screening](#tournament-screening)
 
 In addition to the expectations for programs above, tools or systems used to filter registrants or seed players must also abide by the following practices:
 
-- If a dataset is used, it must be up to date, publicly available and downloadable. Documentation must be provided explaining how to recreate the dataset and reproduce the results using it — including code where applicable.
+- If a dataset is used, it must be publicly available, downloadable, and no more than 30 days out of date. Documentation must be provided explaining how to recreate the dataset and reproduce the results using it — including code where applicable.
```

Comment:
can you give an example of what "dataset" refers to here? data on what? it doesn't mean anything to an outsider

Reply:
Dataset essentially means a database of osu! match data which these programs reference in their algorithms. So the context here is "If the program you want to use has a database it builds its performance network from, it must be public".

I think dataset is used twice here to mean two different things. The first use of dataset is talking about what I write above. The second use is, I think, talking about the actual performance network.

There are community tools out there / in development which convert a large amount of osu! match data into some meaningful measure of performance on a per-player basis. i.e. "Player X has 500 rating and Player Z has 200, therefore Player X should win much more often." This rule is in place to allow third parties to validate that the database of osu! data is valid and that the "rating ladder", if you will, isn't being tampered with. Without this rule, hypothetically, the maintainers of such programs could be artificially manipulating the osu! match data fed into the system. They could, in theory, also be artificially boosting or weakening certain players to manipulate who is eligible for certain tournaments.